packages/graphback-codegen-schema/tests/GraphQLSchemaCreatorTest.ts

```diff
@@ -69,3 +70,28 @@ test('Test snapshot config js', async () => {
   const schema = schemaGenerator.transformSchema(metadata)
   expect(printSchema(schema)).toMatchSnapshot();
 });
+
+test('Test one side relationship schema query type generation', async () => {
```

Comment:
This is good but we already have lots of coverage on this with unit tests.

Reply:
I was just checking that `transformSchema` does work as expected